src/Solvers/WordSearchComponent.tsx

```diff
@@ -0,0 +1,171 @@
+import { WordSearchSolver, WordSearchResult } from 'puzzle-lib';
+import React, { FormEvent, MouseEvent, SyntheticEvent } from 'react';
+import { Button, ButtonGroup, ButtonToolbar, FormControl, Grid, Row, Col } from 'react-bootstrap';
+import LocalStorageComponent from '../Data/LocalStorageComponent';
+import './WordSearchComponent.css';
+
+type Props = {};
+type State = {
+  gridInputText: string,
+  wordListInputText: string,
+  output: JSX.Element[],
+};
+
+type SavedState = {
+  gridInputText: string,
+  wordListInputText: string,
+};
+
+class WordSearchComponent extends LocalStorageComponent<Props, State, SavedState> {
+  private _gridInputText: string = '';
+  private _gridInputElement?: HTMLInputElement;
+  private _wordListInputText: string = '';
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      gridInputText: '',
+      wordListInputText: '',
+      output: [],
+    };
+  }
+
+  public componentDidMount() {
+    super.componentDidMount();
+
+    if (this._gridInputElement) {
+      this._gridInputElement.focus();
+    }
+  }
+
+  public render() {
+    return (
+      <div className="WordSearchGrid">
+        <Grid>
+          <Row>
+            <Col md={4}>
+              <FormControl
+                componentClass="textarea"
+                className="WordSearchList-input"
+                onChange={(event: FormEvent<FormControl>) => this.onListTextChanged(event)}
+                placeholder="Word List To Find"
+                value={this.state.wordListInputText}
+              />
+            </Col>
+
+            <Col md={8}>
+                <FormControl
+                  componentClass="textarea"
+                  className="WordSearchGrid-input"
+                  spellCheck={false}
+                  inputRef={(input: HTMLInputElement) => { this._gridInputElement = input; }}
+                  onChange={(event: FormEvent<FormControl>) => this.onGridTextChanged(event)}
+                  placeholder="Grid Text"
+                  value={this.state.gridInputText}
+                />
+            </Col>
+          </Row>
+
+          <pre className="WordSearchGrid-output">
+            {this.state.output}
+          </pre>
+        </Grid>
+      </div>
+    );
+  }
+
+  protected getLocalStorageKey() {
+    return 'WordSearchGrid';

```

Comment:
Have this match the class name, there just wasn't a good way to get it automatically when the code is minified.